doc/source/tutorial/stats/continuous_landau.rst

```diff
@@ -0,0 +1,16 @@
+
+.. _continuous-landau:
+
+Landau distribution
+===================
+
+A special case of Lévy-stable distributions with :math:`\alpha=1`
+and :math:`\beta=1` and support :math:`-\infty < x < \infty`. The probability
+density function is given by
+
+.. math::
+   :nowrap:
+
+    f(x) = \frac{1}{2\pi i}\int_{a-i\infty}^{a+i\infty}e^{s\log(s)+xs}\ ds
```

Comment:
Unless I hear otherwise, I'll replace this with the next version of the integral on WP, the real one without location/scale.

I'll also add here that the moments are undefined.